File: UnitySokoban/Assets/Scripts/Planning/HeuristicSearchPlannerSGW/HeuristicPlanner.cs
using Planning;
using StateSpaceSearchProject;

namespace HeuristicSearchPlannerSGW
{
    public abstract class HeuristicPlanner : Planner<HeuristicSearch>
    {

        public HeuristicPlanner(string name)
            :base(name)
        {
        }
    }

}

File: Heuristic Search (sgware)/src/edu/uno/ai/planning/h/HeuristicPlanner.java
package edu.uno.ai.planning.h;

import edu.uno.ai.planning.ss.StateSpacePlanner;
import edu.uno.ai.planning.ss.StateSpaceProblem;

public abstract class HeuristicPlanner extends StateSpacePlanner {

	public HeuristicPlanner(String name) {
		super(name);
	}

	@Override
	protected abstract HeuristicSearch makeStateSpaceSearch(StateSpaceProblem problem);
}
